src/environmentd/src/http.rs

```diff
@@ -289,8 +292,17 @@ where
 
     async fn from_request_parts(
         req: &mut http::request::Parts,
-        _state: &S,
+        state: &S,
     ) -> Result<Self, Self::Rejection> {
+        #[derive(Debug, Default, Deserialize)]
+        struct Params {
+            #[serde(default)]
+            options: String,
+        }
+        let params: Query<Params> = Query::from_request_parts(req, state)
```

Comment:
This needs some docs at some point too.

Reply:
Totally, I'll add these in a followup PR